docs/data/DATA.md

```diff
@@ -1,12 +1,12 @@
 # Data
 
-Data is currently generated from a Wikidata JSON dump loaded into Hadoop on the Wikimedia Foundation Analaytics cluster. See DATA-NEW.md
+Data is currently generated from a Wikidata JSON dump loaded into Hadoop on the Wikimedia Foundation Analaytics cluster, see [DATA-NEW.md](DATA-NEW.md).
 
-Old data can also be gathered from the old wikidata-analysis project, See DATA-OLD.md
+Old data can also be gathered from the old wikidata-analysis project, see [DATA-OLD.md](DATA-OLD.md).
```

Comment:
@outdooracorn The tool is actually linked to from the linked to documentation https://github.com/wmde/wikidata-map/blob/master/docs/data/DATA-OLD.md